### DISCUSSION TITLE
When can we expect 1.4.3?

### DISCUSSION THREAD

**Discussion by ptahchiev** [JavaMoney/jsr354-ri#395] [General]
I noticed you have switched to Jakarta EE and now use the `jakarta` package names instead of `javax`. When can we expect the next release?

**Comment by keilw** [comment]
This is related to #383 and #389.
While the ECB rate refresh issues are solved in a decent way, there is no real solution for the IMF problem (#353)

If anybody has capacity to help with #353 please go ahead. That is the only remaining blocker for 1.4.3.
